### ISSUE TITLE
message returned from a failed registration can be None when more is available

### ISSUE THREAD

**Issue by tahoemph** [stormpath/stormpath-flask#24]
On line 88 of flask_stormpath/views.py there is something that looks like

    except StormpathError, err:
        flash(err.user_message)

When inputting a short password to the bare bones implementation you get back

    Error({u'status': 400, u'developerMessage': u'Account password minimum length not satisfied.', u'message': u'Account password minimum length not satisfied.', u'code': 2007, u'moreInfo': u'http://docs.stormpath.com/errors/2007'},)

err.message['message'] contains roughly what you.  I am new to flask-stormpath and am uncertain if this is the right interface or there is something else going on here.  Otherwise I'd just submit a patch.

**Comment by rdegges** [comment]
Thanks for the find! I'm looking into this now, but this appears to be 100% correct.

I'm trying to figure out why I am saying `user_message` here so often -- I'm positive it was working at one point, but, hrm. Maybe I'm just crazy!

**Comment by rdegges** [comment]
Hmm, so I just added additional integration tests to cover this, but here's the weird thing: regardless of what the error object looks like, both `message` AND `user_message` always contain the same thing.

So the actual code as it is works just fine.

I went ahead and fixed this anyhow to use `err.message` instead, as I have no idea why `err.user_message` is working. Super weird.

Anyhow: cutting a release with the updated changes now.
